calico_node/startup/startup.go

```diff
@@ -771,6 +771,11 @@ func ensureDefaultConfig(c *client.Client, node *api.Node) error {
 		return err
 	}
 
+	// Update the ClusterType with the type in CLUSTER_TYPE
```

Comment:
Seems reasonable, but that doesn't match with what was in the Design doc and it doesn't match the key in the datastore.